doc/RELEASE_PROCESS.rst

```diff
@@ -78,69 +74,47 @@ This is an outline of what needs to be done in order to release rst2pdf.
     ::
 
        $ echo -e "[egg_info]\ntag_build=rc1\n" > /tmp/build_opts.cfg
-       $ DIST_EXTRA_CONFIG=/tmp/build_opts.cfg python -m build
+       $ DIST_EXTRA_CONFIG=/tmp/build_opts.cfg uv build
        $ rm /tmp/build_opts.cfg
 
     If you're doing an alphaX, betaX or postX, then change ``tag_build=rc1`` appropriately
 
-#. Set up PyPI if you haven't already
-
-    Create a ``~/.pypirc`` file with sections for rst2pdf and testrstpdf which are then used with ``twine`` via the
-    ``--repository`` switch.
-
-    It should contain the following:
-
-    ::
-
-        [rst2pdf]
-          username = __token__
-          password = {your token here}
-
-        [testrst2pdf]
-          repository = https://test.pypi.org/legacy/
-          username = __token__
-          password = {your token here}
-
-
-    You can get your token from the account settings section of https://pypi.org/ & https://test.pypi.org/
-
-
-#. Upload the rc distribution to Test-PyPI_
+#. Upload the rc distribution to Test-PyPI_. You can get your token from the account settings section of
+   https://test.pypi.org/.
 
     ::
 
-       $ twine upload --repository testrst2pdf dist/*
+       $ uv publish --publish-url https://test.pypi.org/legacy/ --token {your token here}
 
     Check that it all looks correct on Test-PyPI. If not, fix and release a new rc.
 
-#. Test Test-PyPI release into a clean virtual env
+#. Test Test-PyPI release into a clean virtual env (specify the correct version number below). It should install the
+   rc release on Test PyPI and be able to create PDF documents from rst files e.g.
 
     ::
 
-       $ pip install --index-url https://test.pypi.org/simple \
-         --extra-index-url https://pypi.org/simple --pre rst2pdf
-
-    It should install the rc release on Test PyPI and be able to create PDF documents from rst files
+       $ uvx -n --index-url https://test.pypi.org/simple --extra-index-url https://pypi.org/simple --prerelease allow \
+         --index-strategy unsafe-best-match rst2pdf@0.101rc1 tests/input/test_tableofcontents.rst
```

Comment:
Note this example command runs the RC version of `rst2pdf` from Test-PyPI on the `tests/input/test_tableofcontents.rst` test case. I don't have enough experience with `rst2pdf` to know if there's a better choice here, so please feel free to suggest another file.

The previous line only said it should "be able to create PDF documents from rst files", but I think supplying an example here will be more helpful.